acceptance/README.md

```diff
@@ -3,184 +3,219 @@
-    the bolt tests. This value should be expressed in the notation used by
-    `beaker-hostgenerator`. Multiples should be comma separated
-    (i.e.`ubuntu1604-64,osx1012-64`).
-
-SSH_USER
-    :  Value to be used by the tests as the user for connecting to all
-    hosts using the `ssh` role. This user must already be present on the
-    system. If this variable is not provided, it will default to `root`.
-
-SSH_PASSWORD  _required_
-    :  Value to be used by the tests as the passord for the `SSH_USER`
-     for connecting to all hosts over SSH.
-
-WINRM_USER
-    :  Value to be used by the tests as the user for connecting to all
-    hosts using the `winrm` role. This user must already be present on
-    the system. If this variable is not provided, it will default to
-    `Administrator`.
-
-WINRM_PASSWORD  _required_
-    :  Value to be used by the tests as the passord for the `WINRM_USER`
-     for connecting to all hosts over WinRM.
-
-GEM_VERSION
-    :  When testing via gem install, this value will be used by the test
-    pre-suites for the bolt gem version to install on the bolt node. If this
-    variable is not provided, it will default to `> 0.1.0`.
-
-GEM_SOURCE
-    :  When testing via gem install, this value will be used by the test
-    pre-suites for the gem source to be used to install `bolt` from. If this
-    variable is not provided, it will default to `https://rubygems.org`.
-
-GIT_SERVER
-    :  When testing via git install, this value will be used by the test
-    pre-suites in conjunction with `GIT_FORK` to determine where the git repo
-    should be obtained from. If this variable is not provided, it will default
-    to `https://github.com`.
-
-GIT_FORK
-    :  When testing via git install, this value will be used by the test
-    pre-suites in conjunction with `GIT_SERVER` to determine where the git repo
-    should be obtained from. If this variable is not provided, it will default
-    to `puppetlabs/bolt.git`.
-
-GIT_BRANCH
-    :  When testing via git install, this value will be used by the test
-    pre-suites to determine what branch should be checked out for testing.
-    If this variable is not provided, it will default to `master`.
-
-GIT_SHA
-    :  When testing via git install, this value will be used by the test
-    pre-suites to determine what git ref should be checked out for testing.
-    This variable supercedes `GIT_BRANCH` if provided.
-
-SHA
-    :  When testing via package install, this value will be used by the test
-    pre-suites to determine what package version should be installed for
-    testing.
+### Required with no default
+
+**SSH_PASSWORD** (No default): Value to be used by the tests as the
+password for the `SSH_USER` for connecting to all hosts over SSH.
+
+
+**WINRM_PASSWORD** (No default): Value to be used by the tests as the
+passord for the `WINRM_USER` for connecting to all hosts over WinRM.
+
+### Required with default
```

Comment:
`Required with default` is a weird thing to say.

Reply:
Why not just required and optional?

Reply:
Hm, I wasn't sure how to distinguish "This needs to be set, but it's already set for you" and "This doesn't need to be set", but maybe that's not a meaningful distinction?

Reply:
It doesn't feel particularly meaningful.